README.md

```diff
@@ -169,30 +169,23 @@ Configuration information is specified in two forms:
+| `SPRING_SSL_BUNDLE_JKS_WEB_SERVER_KEYSTORE_TYPE`     | `NA`              | (Required). Set the type of the keystore. Common types include `JKS`, `PKCS12`, and `BCFKS` |
+| `SPRING_SSL_BUNDLE_JKS_WEB_SERVER_KEYSTORE_PATH`     | `NA`              | (Required). Set the location of the keystore file in the local file system                  |
+| `SPRING_SSL_BUNDLE_JKS_WEB_SERVER_KEYSTORE_PASSWORD` | `NA`              | (Required). Set the password for the keystore                                               |
 
 ### System Properties for TLS configuration for Web Client
 To configure Transport Layer Security (TLS) for Web clients in the edge module, you can use the following configuration parameters.
 Truststore parameters for configuring Web clients are optional even when ssl_enabled = true.
 If truststore parameters need to be populated, truststore_type, truststore_path and truststore_password are required.
 
-| Property                          | Default           | Description                                                                      |
-|-----------------------------------|-------------------|----------------------------------------------------------------------------------|
-| `web-client.ssl_enabled`          | `false`           | Set whether SSL/TLS is enabled for Vertx Http Server                             |
-| `web-client.truststore_type`      | `NA`              | Set the type of the keystore. Common types include `JKS`, `PKCS12`, and `BCFKS`  |
-| `web-client.truststore_provider`  | `NA`              | Set the provider name of the key store                                           |
-| `web-client.truststore_path`      | `NA`              | Set the location of the keystore file in the local file system                   |
-| `web-client.truststore_password`  | `NA`              | Set the password for the keystore                                                |
-| `web-client.key_alias`            | `NA`              | Set the alias of the key within the keystore.                                    |
-| `web-client.key_alias_password`   | `NA`              | Optional param that points to a password of `key_alias` if it protected          |
+| Property                                | Default           | Description                                                                      |
+|-----------------------------------------|-------------------|----------------------------------------------------------------------------------|
+| `FOLIO_CLIENT_TLS_ENABLED`              | `false`           | Set whether SSL/TLS is enabled for Vertx Http Server                             |
+| `FOLIO_CLIENT_TLS_TRUST_STORE_TYPE`     | `NA`              | Set the type of the keystore. Common types include `JKS`, `PKCS12`, and `BCFKS`  |
+| `FOLIO_CLIENT_TLS_TRUST_STORE_PATH`     | `NA`              | Set the location of the keystore file in the local file system                   |
```

Comment:
we have this spring property: folio.client.tls.trustStorePath
please check does spring requires FOLIO_CLIENT_TLS_TRUST_STORE_PATH env variable or FOLIO_CLIENT_TLS_TRUSTSTOREPATH